modules/server/variables.tf

```diff
@@ -544,19 +557,19 @@ variable "server_boot_image" {
 
 variable "server_instance_metadata" {
   description = "Metadata key/value pairs to make available from within the server instance."
-  type        = "map"
+  type        = map(string)
   default     = {}
 }
 
 variable "server_tags" {
   description = "VM instance tags"
-  type        = "list"
+  type        = list(string)
   default     = []
 }
 
 variable "server_access_config" {
   description = "Server instance 'access_config' block"
-  type        = "map"
+  type        = map(string)
```

Comment:
This should be an object since there is a static set of expected keys.

Reply:
Fixed